models/gyroscope/ethereum/gyroscope_ethereum_trades.sql

```diff
@@ -0,0 +1,56 @@
+{{ config(
+    schema = 'gyroscope_ethereum',
+    alias = 'trades',
+    partition_by = ['block_date'],
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['block_date', 'blockchain', 'project', 'version', 'tx_hash', 'evt_index'],
```

Comment:
you have `version` as a unique key, but it's not in the query of the model, so it's failing looking for it on join conditions for a merge